cache/remote_cache.go

```diff
@@ -8,13 +8,25 @@ import (
 	"github.com/go-kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/grafana/dskit/promregistry"
 )
 
 var (
 	_ Cache = (*MemcachedCache)(nil)
 	_ Cache = (*RedisCache)(nil)
 )
 
+const (
+	labelName             = "name"
+	labelBackend          = "backend"
```

Comment:
[nit] I would clarify it. Especially `labelName` is very generic.

```suggestion
	labelCacheName             = "name"
	labelBackendName          = "backend"
```